remoteworkitem/remoteworkitem_test.go

```diff
@@ -1,18 +1,39 @@
-// +build unit
-
 package remoteworkitem
 
 import (
-	"fmt"
+	"encoding/json"
 	"io/ioutil"
 	"net/http"
 	"testing"
 
+	"github.com/almighty/almighty-core/resource"
 	"github.com/almighty/almighty-core/test"
 )
 
+const (
+	GithubIssueWithAssignee    = "http://api.github.com/repos/almighty-test/almighty-test-unit/issues/2"
+	GithubIssueWithoutAssignee = "https://api.github.com/repos/almighty-test/almighty-test-unit/issues/1"
+	JiraIssueWithAssignee      = "http://jira.atlassian.com/rest/api/latest/issue/JRA-9"
+	JiraIssueWithoutAssignee   = "http://jira.atlassian.com/rest/api/latest/issue/JRA-10"
+)
+
 func provideRemoteGithubData() ([]byte, error) {
-	url := "https://api.github.com/repos/almighty/almighty-core/issues/131"
+	url := GithubIssueWithAssignee
```

Comment:
Most of the code in `provideRemoteGithubData`  and `provideRemoteJiraData` looks identical. Does it makes sense to outsource this into a separate function that accepts a URL and does a HTTP GET and returns a `byte` slice and an error?